src/pages/CommitDetailPage/CommitCoverage/UploadsCard/useUploads/useUploads.js

```diff
@@ -3,20 +3,15 @@ import { useParams } from 'react-router-dom'
 import { useCommit } from 'services/commit'
 import { useRepoOverview } from 'services/repo'
 import { TierNames, useTier } from 'services/tier'
-import { useFlags } from 'shared/featureFlags'
 import { extractUploads } from 'shared/utils/extractUploads'
 
 export function useUploads() {
   const { provider, owner, repo, commit } = useParams()
   const { data: overview } = useRepoOverview({ provider, owner, repo })
-  const { multipleTiers } = useFlags({
-    multipleTiers: false,
-  })
   const { data: tierData } = useTier({ provider, owner })
 
   // TODO: We need backend functionality to properly manage access to carryforward flags for team tier members
-  const isTeamPlan =
-    multipleTiers && tierData === TierNames.TEAM && overview?.private
+  const isTeamPlan = tierData === TierNames.TEAM && overview?.private
```

Comment:
I know your PR is not introducing this concept but I found it confusing that many files here don't separate out like `isTeamPlan` && `isRepoPrivate` and instead put them both into one arguably-misnamed variable. 

Unless I'm misunderstanding what a "teamPlan" means...

Reply:
this is definitely something worth bringing up on slack, its a bit of a black box to me as well